pkg/test-infra/pkg/tidb/operation.go

```diff
@@ -297,6 +299,36 @@ func (t *TidbOps) ApplyDiscovery(tc *v1alpha1.TidbCluster) error {
 	return nil
 }
 
+func (t *TidbOps) GetPDMember(namespace, name string) (string, []string, error) {
```

Comment:
TiDB-cluster track and record the PD leader and member at TC.status.PD.Members, so I use that information to retrieve the leader and members currently.